lib/Bio/EnsEMBL/DataCheck/Checks/MLSSTagMultipleAlignment.pm

```diff
@@ -0,0 +1,96 @@
+=head1 LICENSE
+
+Copyright [2018-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::MLSSTagMultipleAlignment;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::Compara;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME        => 'MLSSTagMultipleAlignment',
+  DESCRIPTION => 'Multiple alignments have appropriate tags',
+  GROUPS      => ['compara', 'compara_genome_alignments'],
+  DB_TYPES    => ['compara'],
+  TABLES      => ['method_link', 'method_link_species_set', 'method_link_species_set_tag']
+};
+
+sub skip_tests {
+  my ($self) = @_;
+  my $mlss_adap = $self->dba->get_MethodLinkSpeciesSetAdaptor;
+  my @methods = qw( EPO EPO_LOW_COVERAGE PECAN );
+  my $db_name = $self->dba->dbc->dbname;
+  
+  my @mlsses;
+  foreach my $method ( @methods ) {
+    my $mlss = $mlss_adap->fetch_all_by_method_link_type($method);
+    push @mlsses, @$mlss;
+  }
+  
+  if ( scalar(@mlsses) == 0 ) {
+    return( 1, "There are no multiple alignments in $db_name" );
+  }
+}
+
+sub tests {
+  my ($self) = @_;
+
+  my $tags = [
+    'num_blocks',
+    'max_align',
+  ];
+
+  has_tags($self->dba, 'EPO', $tags);
+  has_tags($self->dba, 'PECAN', $tags);
+
+  push @$tags, 'base_mlss_id';
+  has_tags($self->dba, 'EPO_LOW_COVERAGE', $tags);
+
+  my $desc = "Low coverage alignments tagged with base MSA MLSS ID";
```

Comment:
No longer "low coverage alignments" - simply "extensions" because the included genomes are no longer necessarily "low coverage".
```suggestion
  my $desc = "Extended multiple sequence alignments tagged with base MSA MLSS ID";
```